src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/featured-image.php

```diff
@@ -23,7 +23,7 @@
 <div class="tribe-events-calendar-month__calendar-event-tooltip-featured-image-wrapper">
 	<a
 		href="<?php echo esc_url( $event->permalink ); ?>"
-		title="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
+		title="<?php echo esc_attr( $event->title ); ?>"
```

Comment:
https://github.com/moderntribe/the-events-calendar/pull/2987/files?utf8=%E2%9C%93&diff=split&w=1#r366022101